go/chat/utils/utils.go

```diff
@@ -889,13 +906,20 @@ func GetDesktopNotificationSnippet(conv *chat1.ConversationLocal, currentUsernam
 	if conv == nil {
 		return ""
 	}
-	msg, err := PickLatestMessageUnboxed(*conv, chat1.VisibleChatMessageTypes())
+	msg, err := PickLatestMessageUnboxed(*conv,
+		append(chat1.VisibleChatMessageTypes(), chat1.MessageType_REACTION))
 	if err != nil || !msg.IsValid() {
 		return ""
 	}
 	mvalid := msg.Valid()
+	var snippet string
 	if !mvalid.IsEphemeral() {
-		snippet, _ := GetMsgSnippet(msg, *conv, currentUsername)
+		switch msg.GetMessageType() {
+		case chat1.MessageType_REACTION:
+			snippet = "reacted to your message."
```

Comment:
Do we want to show what they reacted with?

Reply:
Turned out to be harder than I expected. We store something like :+1: in the actual message text for the reaction, but only the JS code knows how to turn that into an actual emoji. If we like this, we can try and plumb all that through.

Reply:
Ah, gotcha. Knowing how they reacted would be super helpful (so you don't have to go look at the message just to see the emoji), but it's not a deal-breaker.

Reply:
I can trivially put the colon in there, but I wasn't sure if people would like that.

Reply:
we have a library vendored for this actually for the cli https://github.com/keybase/client/blob/master/go/client/chat_cli_rendering.go#L686